packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js

```diff
@@ -24,14 +36,32 @@ export function NativeValue({ data, edit, onClick, onChange, jsonpath }) {
 	if (edit) {
 		return <input type={inputType} value={data} onChange={e => onChange(e, { jsonpath })} />;
 	}
+
+	const isSelectedLine = (selectedJsonpath && (selectedJsonpath === jsonpath));
+
+	function stopAndSelect(event) {
```

Comment:
this function is created on each of its parent eval, maybe you can try to extract it.